samples/cds-sample-application/package.json

```diff
@@ -25,26 +25,26 @@
     "ci-package": "rimraf deployment && mkdir deployment && cp -r dist package.json package-lock.json db srv .cdsrc.json deployment/",
     "cds-build": "cds build/all",
     "cds-deploy": "cds deploy",
-    "generate-client": "npx generate-odata-client --inputDir resources/service-specs --outputDir src/generated --clearOutputDir --generateJs=false --serviceMapping=resources/service-specs/service-mapping.json"
+    "generate-client": "npx generate-odata-client --input resources/service-specs --outputDir src/generated --clearOutputDir --transpile=false --optionsPerService=resources/service-specs/service-mapping.json"
```

Comment:
I think here we should use `options-per-service.json` instead

Reply:
yes